### ISSUE TITLE
Test suite fails when the .git directory is not available

### ISSUE THREAD

**Issue by voxik** [heroku/heroku#1125]
If you remove the .git directory from the root sources (or if you execute the test suite from gem for example), the test suite fails with following errors:

```
Failures:
  1) Heroku::Command::Git remote adds -r remote
     Failure/Error: STDOUT
       expected: "Git remote other added\n"
            got: "" (using ==)
       Diff:
       @@ -1,2 +1 @@
       -Git remote other added
     # ./spec/heroku/command/git_spec.rb:127:in `block (3 levels) in <module:Command>'
  2) Heroku::Command::Git remote adds remote
     Failure/Error: STDOUT
       expected: "Git remote heroku added\n"
            got: "" (using ==)
       Diff:
       @@ -1,2 +1 @@
       -Git remote heroku added
     # ./spec/heroku/command/git_spec.rb:115:in `block (3 levels) in <module:Command>'
```

As a workaround, it is enough to do ```git init``` in the root of the gem.

**Comment by voxik** [comment]
This seems to be resolved testing with 3.30.3.

**Comment by dickeyxxx** [comment]
great!
